### ISSUE TITLE
Paragraphs instead of lines in gpageseg

### ISSUE THREAD

**Issue by xdsv** [tmbdev/ocropy#228]
Is there a way to get paragraphs instead of lines in ocropy-gpageseg?
If that's not possible, from what I saw only the .pseg file contains the paragraph information encoded as a different color. Is there a utility to extract that information?

**Comment by zuphilip** [comment]
No, the page segmentation will always output lines. However during this process it tries to identify the columns (I am not sure that also paragraphs are identified in this step) and this information is saved in the pseg files, as you noted already. See [in the wiki for more information about the file format](https://github.com/tmbdev/ocropy/wiki/OCRopus-File-Formats#physical-layout).

Maybe try also to go further and create a hocr file with `ocropus-hocr` which should contain some paragraphs splits as `<p />`.

**Comment by xdsv** [comment]
Thanks for the reply,

actually what I'm looking for is find blocks of text in an image. Can
ocropy do this?

**Comment by Sultan91** [comment]
Also interested in this question. In addition, it is not quite clear how text line ordering works
